clients/fides-js/src/lib/tcf/utils.ts

```diff
@@ -24,6 +25,16 @@ export const transformFidesStringToCookieKeys = (
       cookieKeys[cookieKey] = items;
     }
   });
+
+  // Set AC consents, which will only be on vendor_consents
```

Comment:
I'm actually not sure if this is true. Will AC consents only apply to `vendor_consents`? This might not be true if an AC vendor was assigned a legitimate interest purpose—is that a state we can expect? 

if so, this big of logic gets more complicated to implement because at this point we don't have the experience yet—we can't tell if the AC string is talking about consent or legitimate interest (unless AC is by definition, consent only?)

Reply:
I'm glad you raised this because I likely need to revisit on the backend too for related reasons.

When someone hands the backend an AC string with which to save preferences, I am saving everything under vendor consents, while it's technically possible an AC system shows up in the Vendor Legitimate Interests section too https://github.com/ethyca/fides/pull/4266#discussion_r1357097200

Reply:
Yes, just vendor consents - it's explicitly _only_ for consent basis, that's a very reasonable assumption to make here.

Reply:
Updating here Allison, I now also prevent AC systems from showing up in the Vendor Legitimate interests section of the experience too so everything matches

Reply:
thanks, both! will keep this logic as-is then :)